client/src/document/ingredients/browser-compatibility-table/index.scss

```diff
@@ -140,6 +152,32 @@
 .bc-version-label {
   display: none;
 }
+.expert .bc-version-label {
+  display: initial;
+}
+
+.expert .bc-icon-cell {
+  > .icon-wrap {
+    display: block;
+  }
+}
+
+.expert {
```

Comment:
Instead of `.expert`, I would suggest a self-descriptive name like `.with-versions`.

This also avoids the notion that only "experts" need the version numbers, or that you're not an "expert", unless you care about version numbers.